babel.config.js

```diff
@@ -2,15 +2,7 @@ module.exports = {
   env: {
     test: {
       presets: [['@babel/preset-env', { targets: { node: 'current' } }]],
-      plugins: [
-        ['@babel/plugin-transform-react-jsx'],
-        [
-          'istanbul',
-          {
-            exclude: ['**/test/**', '**/__test__/**', '**/dist/**'],
-          },
-        ],
-      ],
+      plugins: ['@babel/plugin-transform-react-jsx'],
```

Comment:
why remove these?

Reply:
because my guess is that jest is automatically covers the that for code coverage and istanbul settings. i think `aw` has problems with it and it was required for `aw` in order to report the coverage, but jest does it all for us.

If i keep it, i'm getting below error for duplicate plugin/preset error:

<img width="1472" alt="Screenshot 2022-09-23 at 10 10 37" src="https://user-images.githubusercontent.com/29652890/191918032-e781cf56-9298-4f98-ae70-41f86b7870c9.png">